src/components/TalkDetail.astro

```diff
@@ -0,0 +1,115 @@
+---
+import { graphql, FragmentOf } from "gql.tada";
+import Markdown from "./Markdown.astro";
+
+const formatDate = (date: string) => {
+	return new Date(date).toLocaleDateString("en-US", {
+		weekday: "long",
+		month: "short",
+		day: "numeric",
+	});
+};
+
+export const talkDetailFragment = graphql(`
+	fragment TalkDetailFragment on Talk @_unmask {
+		title
+		abstract
+		room
+		date
+		fromTime
+		toTime
+		tagsCollection {
+			items {
+				title
+			}
+		}
+	}
+`);
+
+export interface Props {
+	talk: FragmentOf<typeof talkDetailFragment>;
+}
+
+const { talk } = Astro.props;
+---
+
+<article class="talk">
+	<h3>{talk.title}</h3>
+	<div class="meta">
+		<div class="metaGroup">
+			<div class="talkType typo-body-caption">Talk</div>
+			<div>{formatDate(talk.date)}</div>
```

Comment:
As we don't know the day yet for talks, this would have to be optional, too.

Reply:
done